pyblish/cli.py

```diff
@@ -303,4 +334,42 @@ def publish(ctx,
         click.echo(_format_time(_start, _end))
 
 
+@click.command()
+@click.argument("package", default="pyblish_qml")
+@click.pass_context
+def gui(ctx, package):
+
+    environ = os.environ.copy()
+    context = ctx.obj["context"]
+
+    with _cli_plugin(data=context.data) as plugin_path:
+        environ["PYBLISHPLUGINPATH"] = os.pathsep.join(
+            ctx.obj["plugin_paths"] + [plugin_path]
+        )
+
+        process = subprocess.Popen(
+            [sys.executable, "-m", package + ".__main__"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
```

Comment:
Tip: If you make this `subprocess.STDOUT` instead, errors will be passed through stdout, and you'll only need to look for output there.